### PR TITLE
Add CSSURLImageValue object

### PR DESCRIPTION
**Pull request by anthony-huang**

### CODE REVIEW

File: test/js/css-url-image-value.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      assert.strictEqual(urlImageValue.state, "loaded");
+      assert.strictEqual(urlImageValue.intrinsicWidth, 1);
+      assert.strictEqual(urlImageValue.intrinsicHeight, 1);
+      assert.strictEqual(urlImageValue.intrinsicRatio, 1);
+      done();
+    }
+  });
+
+  test('Invalid image will have error state and null intrinsic dimensions', function(done) {
+    var urlImageValue = new CSSURLImageValue('http://localhost');
```
Comment (wilddamon) [inline test/js/css-url-image-value.js]: Just use something you know definitely won't return an image. (e.g. resources/nonexisting.png?) Who knows what localhost could return on someone else's machine!

File: test/js/css-url-image-value.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+suite('CSSURLImageValue', function() {
```
Comment (wilddamon) [inline test/js/css-url-image-value.js:1]: The value of cssText isn't being tested anywhere.

Comment (anthony-huang) [reply test/js/css-url-image-value.js:1]: Done.

File: test/js/css-url-image-value.js (hunk 1 above)
Comment (anthony-huang) [reply test/js/css-url-image-value.js]: Done.

**Comment by wilddamon** [comment]
LGTM

**Pull request merged**